launch/client.py

```diff
@@ -1416,6 +1279,768 @@ def get_batch_async_response(self, batch_job_id: str) -> Dict[str, Any]:
+                gpu_type=GpuType(gpu_type) if gpu_type is not None else None,
+                max_workers=max_workers,
+                memory=memory,
+                min_workers=min_workers,
+                model_bundle_id=model_bundle_id,
+                per_worker=per_worker,
+                post_inference_hooks=post_inference_hooks or [],
+                default_callback_url=default_callback_url,
+                storage=storage,
+            )
+            update_model_endpoint_request = UpdateModelEndpointRequest(
+                **payload
+            )
+            path_params = frozendict({"model_endpoint_id": model_endpoint_id})
+            response = api_instance.update_model_endpoint_v1_model_endpoints_model_endpoint_id_put(  # type: ignore
+                body=update_model_endpoint_request,
+                path_params=path_params,  # type: ignore
+                skip_deserialization=True,
+            )
+            return json.loads(response.response.data)
+
+    def list_model_endpoints_v1(self) -> List[Dict[str, Any]]:
+        """
+        Lists all model endpoints that the user owns.
+
+        Returns:
+            A list of ``ModelEndpoint`` objects.
+        """
+        with ApiClient(self.configuration) as api_client:
+            api_instance = DefaultApi(api_client)
+            response = (
+                api_instance.list_model_endpoints_v1_model_endpoints_get(
+                    skip_deserialization=True
+                )
+            )
+            return json.loads(response.response.data)
+
+    def get_model_endpoint_by_id_v1(
+        self, model_endpoint_id: str
+    ) -> Dict[str, Any]:
+        """
+        Gets a model endpoint for a given ID.
+
+        Parameters:
+            model_endpoint_id: The name of the endpoint to retrieve.
+        """
+        with ApiClient(self.configuration) as api_client:
+            api_instance = DefaultApi(api_client)
+            path_params = frozendict({"model_endpoint_id": model_endpoint_id})
+            response = api_instance.get_model_endpoints_api_v1_model_endpoints_api_get(  # type: ignore
+                path_params=path_params,
+                skip_deserialization=True,
+            )
+            return json.loads(response.response.data)
+
+    def get_model_endpoint_by_name_v1(
+        self, model_endpoint_name: str
+    ) -> Dict[str, Any]:
+        """
+        Gets a model endpoint for a given ID.
+
+        Parameters:
+            model_endpoint_name: The name of the endpoint to retrieve.
+        """
+        with ApiClient(self.configuration) as api_client:
+            api_instance = DefaultApi(api_client)
+            query_params = frozendict({"name": model_endpoint_name})
+            response = api_instance.get_model_endpoints_api_v1_model_endpoints_api_get(  # type: ignore
+                query_params=query_params,
+                skip_deserialization=True,
+            )
+            return response
```

Comment:
need to parse

Reply:
I skipped serialization because there are sometimes inconsistencies between return types and the auto-generated response types. They're finicky sometimes and imo the extra typing we get isn't worth the hassle.